pyquil/device.py

```diff
@@ -121,17 +110,16 @@ def from_dict(d):
         :rtype: ISA
         """
         return ISA(
-            name=d["id"]["name"],
-            version=d["id"].get("version", "0.0"),
-            timestamp=d["id"].get("timestamp"),
-            qubits=[Qubit(id=q["qubit-id"],
-                          type=q.get("type", DEFAULT_QUBIT_TYPE),
-                          dead=q.get("dead", False))
-                    for q in d["logical-hardware"][0]],
-            edges=[Edge(targets=e["action-qubits"],
-                        type=e.get("type", DEFAULT_EDGE_TYPE),
-                        dead=e.get("dead", False))
-                   for e in d["logical-hardware"][1]],
+            qubits=sorted([Qubit(id=int(qid),
```

Comment:
does sorting these matter? (no matter your answer, i'm not sure a change is necessitated; i'm just confused.)

Reply:
@ecp-rigetti It matters for `test_isa`. As written, to show equality that test does require an equivalent ordering for both `qubits` and `edges`. Alternatively we could change the test.